lib/portlayer/network/context.go

```diff
@@ -68,9 +68,28 @@ func NewContext(bridgePool net.IPNet, bridgeMask net.IPMask) (*Context, error) {
 	if err != nil {
 		return nil, err
 	}
-
 	s.builtin = true
 	ctx.defaultScope = s
+
+	// add any external networks
+	for nn, n := range portlayer.Config.ContainerNetworks {
+		if nn == "bridge" {
```

Comment:
With my current PR the tether takes care of bringing the bridge interface up and setting it's IP - does the PL need to do specific handling for it, or can it be considered just another container network at this point?
(not considered the port forwarding with this comment).